### DISCUSSION TITLE
Smaller Ui

### DISCUSSION THREAD

**Discussion by Jebs88** [keiwando/toonsquid#650] [Ideas]
Hoping for a smaller Ui, like in Affinity Designer.

Can't wait for v2. *just showing my excitement. 🥹😅

**Comment by otaworks** [comment]
could you please show a picture? I don't use Affinity yet, so I'm not sure of how it looks like

**Reply by Jebs88** [reply]
It's not that much but still smaller.


![IMG_2641](https://github.com/user-attachments/assets/47081429-60be-4fe7-8502-aa1bf1295efc)
![IMG_ 2640](https://github.com/user-attachments/assets/f9015120-2671-44df-8bd1-b896ec44caa2)

**Reply by otaworks** [reply]
()Okay, I get it now. I also prefer a smaller, minimalist look. Hopefully this becomes an option to choose either a big or smaller interface to work with. I also posted with a different account how I hope the timeline would give the option to turn off the thumbnail on the timeline, so we could see more layers and have a more simple overview of everything.

**Reply by Jebs88** [reply]
Yes, this would be great for the mini users.

**Comment by TempleLionStudios** [comment]
Pleeease no cluttered interface! Procreate does this well.

I really like the interface in TS rn. Timeline could be cool to see smaller tho. 

I share your excitement too about 2.0! 🥳🤩 

Thanks for all the work you do Keiwan! ♥️🫶🏼

**Reply by Jebs88** [reply]
A customisable toolbar would be nice, like what Noteful did.

Right?! He has already done wonders.
